packages/ws-server/src/webSocketServer.ts

```diff
@@ -213,6 +215,10 @@ app.ws.use(async (ctx) => {
 
         ctx.websocket.send(JSON.stringify(response));
     });
+
```

Comment:
should we make it a configurable property, if `pingInterval > 0`. ? 
Supposed we fixed it from the infrastructure side, or a different deployment setup does not need it.